src/components/Trigger/Trigger.tsx

```diff
@@ -14,6 +14,11 @@ export enum TriggerSize {
     Large = 'Large',
 }
 
+export enum TriggerEmphasis {
```

Comment:
Instead of having 2 types with the same values, I would create a general one `InputEmphasis`, as this new property is going to be used in `Combobox`, `Select` and `MultiSelect`

Reply:
Changed 👍🏻